src/common/components/addressForm/addressForm.component.spec.js

```diff
@@ -114,4 +115,14 @@ describe('addressForm', function () {
       expect(self.controller.loadingRegionsError).toEqual(true)
     })
   })
+
+  describe('onPostalCodeChanged', () => {
+    it('should trigger onAddressChanged callback', () => {
+      self.controller.countries = countriesResponse._element
+      jest.spyOn(self.controller, 'onAddressChanged')
+      self.controller.onPostalCodeChanged()
+
+      expect(self.controller.onAddressChanged).toHaveBeenCalledWith()
```

Comment:
I developed a philosophy from working in other JavaScript projects that I want to make the strongest assertion possible.  Here I want to assert two things: 1) that it was called, and 2) the inputs for the method call are correct.  `toHaveBeenCalledWith()` does both, while `toHaveBeenCalled()` does only one.

In contrast, if I want to assert that something was not called, I exclusively use `.not.toHaveBeenCalled()` over `.not.toHaveBeenCalledWith()`.  The latter is not strong enough of an assertion, because the method might in fact be called with different inputs and we would not know.  `.not.toHaveBeenCalled()` is the stronger assertion in this case.

I eventually just developed a practice of using `.toHaveBeenCalledWith()` and `.not.toHaveBeenCalled()`.  That being said, if you think it's better that I use `toHaveBeenCalled()` here, I'm fine making that change.